web-app/src/components/BookedLeave/container.js

```diff
@@ -34,13 +38,71 @@ export default Wrapped =>
         });
     }
 
+    canCancelHoliday = (holiday) => {
```

Comment:
It'd be nicer to have this arrow function style for `cancelHoliday` and `handleCancelHoliday` as well - it would get rid of the icky binding in the constructor :)

Reply:
and that's done and pushed :-)

Reply:
👍